### PR TITLE
Assignment   react implementation

### PR DESCRIPTION
**Pull request by Louisdrejer**
assignment 08-11-2023

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,12 +1,255 @@
...
+  transform: rotate(45deg);
+  margin-left: 20px;
+  margin-bottom: 3px;
+}
+.menubox{
+  display: flex;
+  padding: -1px;
+}
+
+.submitButton{
```
Comment (igawaclawska) [inline src/App.css:182]: Consider naming CSS selectors using kebab-case.
https://www.freecodecamp.org/news/css-naming-conventions-that-will-save-you-hours-of-debugging-35cea737d849/

File: src/Components/LeftNav.jsx [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    if (e.key === 'Enter') {
+      console.log(searchTerm);
+    }
+  };
+
+
+  return (
+    <div className="Leftcontainer">
+      <div className="logo_name_left">
+        <img src={logo} alt="Logo" style={{ maxWidth: "70%", marginLeft: "15%" }} />
```
Comment (igawaclawska) [inline src/Components/LeftNav.jsx]: Please avoid inline styling.

File: src/Pages/Login/Login2.jsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import { Link } from "react-router-dom";
+
+import "./Login2.css"
```
Comment (igawaclawska) [inline src/Pages/Login/Login2.jsx]: Why does Login have two separate files? Login.jsx and Login2.jsx ?

File: src/img/something.js [hunk 4]
```diff
@@ -0,0 +1 @@
+
```
Comment (igawaclawska) [inline src/img/something.js]: Redundant file

File: src/Pages/SearchPage/SearchPage.jsx [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+  };
+
+  const handleLanguageChange = (language) => {
+    setSelectedLanguage(language);
+  };
+
+  const handleSkillLevelChange = (skillLevel) => {
+    setSelectedSkillLevel(skillLevel);
+  };
+
```
Comment (igawaclawska) [inline src/Pages/SearchPage/SearchPage.jsx:24]: Good job on naming these handlers!

File: src/App.jsx [hunk 6]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import './App.css';
+import LeftNav from './Components/Homepage/LeftNav';
+import SubmitBox from './Components/Homepage/SubmitBox';
+import Homepage from './Components/Homepage/Homepage';
+
+function App() {
```
Comment (igawaclawska) [inline src/App.jsx]: I see 2 App files (App.js and App.jsx). 
Are both of them needed? If not - remove the redundant one.

File: src/Pages/Answerpage/Answerpage.jsx [hunk 7]
```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import "../Homepage/Homepage.css"
+import SubmitBox from '../../Components/SubmitBox';
+
+export default function Homepage() {
```
Comment (igawaclawska) [inline src/Pages/Answerpage/Answerpage.jsx:5]: Answerpage.jsx looks identical to Homepage.jsx
What about using just one of them?

File: src/Pages/Login/Login2.css [hunk 8]
```diff
@@ -0,0 +1,93 @@
+/*.mainContainerLogin2 {
```
Comment (igawaclawska) [inline src/Pages/Login/Login2.css]: If a file or commented-out code is no longer needed, consider removing it.

**Review by igawaclawska — CHANGES_REQUESTED** [review]
Overall looks good, but please take a look at the comments and try to implement changes. Ping us once you're done. :)

**Comment by mircealungu** [comment]
Hey people, we see that this is merged, while the state of the branch is **changes requested**. You have to make it clear that you have addressed the changes before merging. Either individually answering the comments, or adding a single final comment saying that you've handled everything. So, what happened here?

**Pull request merged**